src/ctapipe/calib/camera/tests/test_calibrator.py

```diff
@@ -130,6 +132,21 @@ def test_check_dl0_empty(example_event, example_subarray):
     assert (event.dl1.tel[tel_id].image == 2).all()
 
 
+def test_dl1_variance_calib(example_event, example_subarray):
+    # test the calibration of variance images
+    tel_id = list(example_event.r0.tel)[0]
+    calibrator = CameraCalibrator(
+        subarray=example_subarray,
+        image_extractor=VarianceExtractor(subarray=example_subarray),
+        image_calibration_type="variance",
+        apply_waveform_time_shift=False,
+    )
+    calibrator(example_event)
+    image = example_event.dl1.tel[tel_id].image
+    assert image is not None
+    assert image.shape == (1764,)
```

Comment:
from where this shape comes? What telescope is this?

Reply:
That is from the example event. Not sure what telescope this is

Reply:
FlashCam from the number of pixels